tools/odbc/test/isql-test.py

```diff
@@ -171,6 +171,8 @@ def test(cmd, out=None, err=None, extra_commands=None, input_file=None):
 test("SELECT TIMESTAMP '-8-01-01 00:00:01.5'::VARCHAR", out='0009-01-01 (BC) 00:00:01.5')
 test("SELECT TIMESTAMP '100000-01-01 00:00:01.5'::VARCHAR", out='100000-01-01 00:00:01.5')
 
+test("UPDATE timestamp SET t = strptime('20221215101010','%Y%m%d%H%M%S') WHERE t='2008-01-01 00:00:01'::TIMESTAMP")
```

Comment:
Do we want a test for the result?

Reply:
Dunno - I was kind of winging it here. Mostly I just wanted the test to succeed as I can't seem to verify it on my Mac!